audittail-helm-library/tests/consumerapp/templates/deployment.yaml

```diff
@@ -16,13 +16,22 @@ spec:
     spec:
       initContainers:
         {{- include "audittail.initContainer" .| nindent 8 }}
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
```

Comment:
Could we make these the default and not rely on the user setting this? The idea is to promote good security practices.